clients/fossid-webapp/src/test/kotlin/FossIdClientReturnTypeTest.kt

```diff
@@ -122,6 +124,23 @@ class FossIdClientReturnTypeTest : StringSpec({
         }
     }
 
+    "Snippets can be listed when there is some" {
```

Comment:
Maybe shorter: "Existing snippets can be listed".

Reply:
But I see other tests should then be reworded accordingly as well.

Reply:
Can we leave them as if ? I don't intend to reword all tests now.